text/0000-add-geo-processing-abilities-to-skylark.md

```diff
@@ -0,0 +1,411 @@
+- Feature Name: Add geo processing abilities to skylark 
+- Start Date: 2018-10-02
+- RFC PR: <!-- (leave this empty) -->
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+
+# Motivation
+[motivation]: #motivation
+
+A lot of open public data, environmental data, government data etc has some kind of geospatial properties. Being able to 
+perform geospatial operations on these datasets would be a huge win for qri and could help increase adoption of qri as a means
+to distribute these types of dataset
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+When working with spatial datasets, those associated with a point, line, polygon etc we often need to be able to perform manipulations
+of the data based on those geometries. The Geo module of sklark provides the basic data types and methods to be able to do just this.
+Based on the python [Shapley](https://pypi.org/project/Shapely/) library the Geo module allows you to perform operations such as 
+
+- Load Points Lines and Polygons in GeoJSON format and represent these as types withing qri
+- Calculate distances between geometric features 
+- Perform filtering based on geometric conditions such as Within, Intersects, Contains etc 
+- Manipulate lines and polygons with operations like intersects, union etc.
+
+Some use-cases where this would be useful would be 
+
+- Working with 311 complaint data from a open government data portal and wanting to produce a summary of counts of complaint by type in each census tract. This would require grabbing data from the dataset but also the census tracts for an area and performing a spatial join and aggregation on that data.
+require performing a spatial join between the 311 data and a census tract polygon layer for the same region 
+- Augmenting a dataset of bird sightings with the distance to the nearest wetlands.
+- Calculating the overlap of flood regions and building footprints within a region to assess risk
+
+An example of this might look like:
+
+```python 
+use('geo')
```

Comment:
`load` is used for loading modules. It would look like this:
```
load("geo.sky", "geo")
```

Generally speaking, the current docs are actually pretty up to date with the state of the world:
http://qri.io/docs/tutorials/skylark_transformations/